README.md

```diff
@@ -1 +1,40 @@
-# my-to-do-list-
\ No newline at end of file
+# my-to-do-list-
+keep track of your tasks with this to do list 
+
+## DESCRIPTION
+build a simple HTML list of To Do tasks.
+
+## BUILT WITH
+- *Visual Studio Code*
+- *ES6*
+- *Linters for HTML/CSS/JS*
+- *JavaScript*
+- *CSS*
+- *Html*
+- *Webpack*
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps**.
+
+### Prerequisites
+
+- You need to have a code editor to be able to run the code. VSCode is most preferred.
+
+### Setup
+
+- To clone the code press in the green button at the top right corner in this page
+- Select the method that you want to clone
+
+### Usage
+
+- Run npm to install dependencies on webpack => npm i
+- for you to see the actual project working in the browser run npm dev => npm run dev or npm run start
+- if you make some modifications to the project run npm run build in order to bundle your changes on dist folder => npm run build 
+
+### Author
+
+Andres Garzon Maldonado
+    
+  [GITHUB](https://github.com/andgarzonmal)
+  [LINKEDIN](https://www.linkedin.com/in/andres-garzon-maldonado-951a2a180/)
```

Comment:
- [ ] [OPTIONAL] It would be nice to add the issues section, allowing other developers to offer suggestions to improve the project.